src/serac/numerics/functional/interior_face_integral_kernels.hpp

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) 2019-2024, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+#pragma once
+
+#include <array>
+
+#include "serac/serac_config.hpp"
+#include "serac/numerics/functional/quadrature_data.hpp"
+#include "serac/numerics/functional/differentiate_wrt.hpp"
+
+// TODO REMOVE AFTER DEBUGGING
+#include "serac/infrastructure/mpi_fstream.hpp"
```

Comment:
Another one to remove ... sorry, you probably don't need a reminder on each one.

Reply:
these reminders are helpful